dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/EnvironmentService.java

```diff
@@ -50,7 +50,7 @@ public interface EnvironmentService {
      *
      * @param code environment code
      */
-    Map<String, Object> queryEnvironmentByCode(Long code);
+    Result queryEnvironmentByCode(Long code);
```

Comment:
```suggestion
    Environment queryEnvironmentByCode(Long code);
```

Please remove the Result in service.